tests/test_min_username_length.py

```diff
@@ -0,0 +1,24 @@
+import os
+import re
+
+import pytest
+from redis import Redis
+
+from emojyo.main import create_user, get_yos, send_yo
+from emojyo.data import Database
+
+DB = Database()
```

Comment:
```suggestion
from emojyo.main import create_user
```
Importing too many things